app/src/main/java/Model/Apartment.java

```diff
@@ -156,8 +191,40 @@ public int decrementNumberOfResidents() {
         return getNumberOfResidents();
     }
 
-
     /**
      * Sets address fields for the apartment
      */
+    /**
+     * Accessor for the inventory of this apartment.
+     * @return
+     */
+    public Inventory getInventory() {
+        return (Inventory) getParseObject("inventory");
+    }
+
+    /**
+     * Getter for the people who live here.
+     * @return The list of people who live in this apartment.
+     */
+    public ArrayList<Person> getMembers() {
+        return members;
+    }
+
+    public void setStreet_1( String street_1) {
```

Comment:
The thought occurred to me, but I didn't think I like how `street_1` sticks out more than `street1` when glancing through the file, so I went with that.